kong/constants.lua

```diff
@@ -277,7 +277,9 @@ local constants = {
       exit = "kong",
       service = "upstream",
     }
-  }
+  },
+
+  WASM_BUNDLED_FILTERS_PATH = "/usr/local/kong/include/filters",
```

Comment:
bikeshedding: err, `/include/` feels weird as that screams "C headers". Can we use `kong/lib/filters` if we have to use a Unixy dirname? (Or straight `/usr/local/kong/filters` if we don't)

Reply:
:+1: Yeah this is sort of a scratch value that was chosen because it resembled some known paths. We'll pick a more appropriate directory before merge.

Reply:
^ cc @curiositycasualty